docs/source/algorithms/EnggCalibrate-v1.rst

```diff
@@ -110,9 +110,9 @@ Output:
 .. testoutput:: ExampleCalib
 
    DIFA1: 0.00
-   DIFC1: 18400.19
+   DIFC1: 18400.2
    TZERO1: -2.06
    The output table has 1 row(s)
-   Parameters from the table, DIFC1: 18400.19, ZERO1: -2.06
+   Parameters from the table, DIFC1: 18400.2, ZERO1: -2.06
```

Comment:
The changes in `EnggCalibrate` look more than acceptable. This is just a very rough/approximate calculation for the doc-test example. Feel free to tweak the values as needed.
If these are the only unit tests that need updates for GSL2 it seems that we got lucky!